admin/src/lang.json

```diff
@@ -692,32 +692,70 @@
+        "tagPrompt": "إضافة علامات هنا...",
+        "editPrompt": "تحرير الجلسة",
+        "topPrompt": "تحريك إلى أعلى",
+        "mergePrompt": "دمج مع الجلسة …",
+        "whatsNew": "ما <strong style='color: salmon'>الجديد</strong>",
+        "export": "تصدير",
+        "import": "استيراد",
+        "cancel": "الغاء",
+        "exportTitle": "Tab Space علامات التبويب التي تم اخراجها",
+        "exportHTML": "استخراج بلغة HTML",
+        "exportJSON": "تصدير ملف النسخة الاحتياطية",
         "exportText": "نسخ كنص",
-        "exportMD": "Markdown نسخ",
-        "loseTabs": "هل فقدت علامات التبويب الخاصة بك؟ ربما تمت ترقية Tab Space للتو ، يرجى الانتظار قليلاً  والتحديث",
+        "exportMD": "نسخ  Markdown",
+        "loseTabs": "هل فقدت تبوبياتك؟ ربما قد تم ترقية Tab Space، انتظر برهة وقم بإعادة تحديث…",
+        "manuallyMigrate": "انتقال يدوي للجلسات لما قبل Tab Space 3.5",
+        "migrateTip": "أذهب إلى إعدادات ويدويا دمج انتقال الجلسات القديمة",
+        "tips": "تلميحات:",
+        "tipContents": [
+            "إذا ضغطت على \"save tabs\" ووجدت أنه لم يتم حفظ أي علامات تبويب، فذلك بسبب أن تم ترقية Tab Space للتو، يرجى إعادة تشغيل Safari لتفعيله. ",
+            "جلساتك الان تم تزامنها مع كل أجهزتك عبر سحابة اي كلود (macOS 10.15 يتطلب)",
+            "يمكنك الان استخدام السحب والإفلات لإعادة تنظيم الجلسات أو المواقع.",
+            "قم بعمل تقييم Tab Space على استور ابل اذا احببته",
+            "الاختصارات Tab Space لا تعمل عندما يكون شريط عنوان متصفح سفاري فارغ. لأن شريط العنوان الفارغ يعني صفحة خاصة لا يمكن تحميل الأداة فيها."
+        ],
+        "loading": "جاري التحميل …",
+        "ignore-pinned-tabs": "تجاهل صفحات الويب المثبتة أثناء حفظ علامات التبويب",
+        "ignore-duplicated-tabs": "لا تحفظ علامات التبويب المكررة (حتى لمن تلك في جلسات مختلفة)",
+        "save-all-windows": "حفظ علامات التبويب من جميع النوافذ",
+        "shift-shortcuts": "تشغيل الاختصارات باستخدام Ctrl + Shift",
+        "disable-shortcuts": "تعطيل الاختصارات",
+        "disable-context-menus": "تعطيل قوائم السياق",
+        "remove-site-after-click": "قم بإزالة علامة التبويب المحفوظة من الجلسة عند النقر فوقها",
+        "shortcuts": "اختصارات",
+        "shortcutTip": "ملاحظة: الاختصارات لا تعمل عندما يكون شريط عنوان متصفح سفاري فارغ.",
         "ctrlL": "إغلاق علامات التبويب اليسرى",
         "ctrlR": "إغلاق علامات التبويب اليمنى",
         "ctrlQ": "إغلاق كافة علامات التبويب الأخرى",
-        "ctrlC": " chrome افتح في",
-        "ctrlW": "Edge افتح في",
-        "ctrlF": "Firefox افتح في",
-        "ctrlI": "Brave افتح في",
-        "ctrlT": "Tab Space افتح"
+        "ctrlC": "فتح في Chrome",
+        "ctrlW": "فتح في Edege",
+        "ctrlF": "فتح في Firefox",
+        "ctrlI": "فتح في Brave Browser",
+        "ctrlT": "فتح Tab Space",
+        "ctrlB": "Save selected text to Bear",
+        "goTip": "إذا كنت تريد فقط تعيين Tab Space كصفحة افتراضية لعلامة تبويب جديدة ، فيرجى استخدام العنوان \ "http://mytab.space/redirect.html\".",
```

Comment:
Thanks for your contribution!  Just the whitespace in `\ "` needs to be deleted. 😄

Reply:
Got it, thanks